roles/openshift_repos/meta/main.yml

```diff
@@ -11,4 +11,5 @@ galaxy_info:
     - 7
   categories:
   - cloud
-dependencies: []
+dependencies:
+- role: openshift_inventory
```

Comment:
http://docs.ansible.com/ansible/playbooks_roles.html#role-dependencies says "By default, roles can also only be added as a dependency once - if another role also lists it as a dependency it will not be run again."